src/ape/utils.py

```diff
@@ -142,42 +146,43 @@ def compute_checksum(source: bytes, algorithm: str = "md5") -> str:
     return hasher(source).hexdigest()
 
 
-def get_tx_error_from_web3_value_error(web3_value_error: ValueError) -> TransactionError:
+def generate_dev_accounts(
```

Comment:
Some providers need to also generate the dev account while some can simply supply a mnemonic.
Both eth-tester and geth will use this.